models/issue_comment.go

```diff
@@ -389,6 +397,32 @@ func (c *Comment) LoadLabel() error {
 	return nil
 }
 
+// LoadProject if comment.Type is CommentTypeProject, then load project.
+func (c *Comment) LoadProject() error {
+
+	if c.OldProjectID > 0 {
+		var oldProject Project
+		has, err := x.ID(c.OldProjectID).Get(&oldProject)
+		if err != nil {
+			return err
+		} else if has {
+			c.OldProject = &oldProject
+		}
```

Comment:
else?

Reply:
else OldProject is deleted and cant be loaded so leave it nil